crux-mir/test/conc_eval/drop/needs_drop.rs

```diff
@@ -0,0 +1,22 @@
+#![feature(core_intrinsics)]
+
+use core::intrinsics::needs_drop;
```

Comment:
To fix the test suite error that the CI caught, it might be more direct just to do:


```suggestion
use std::intrinsics::needs_drop;
```